openshift_pipelines/installing-pipelines.adoc

```diff
@@ -0,0 +1,19 @@
+[id="installing-pipelines"]
+= Installing OpenShift Pipelines
+include::modules/common-attributes.adoc[]
+include::modules/openshift-pipelines-attributes.adoc[]
+:context: installing-pipelines
+
+toc::[]
+
+// Installing Openshift Pipelines Operator
```

Comment:
Screenshot might be useful similar to these:
https://openshift.github.io/pipelines-docs/docs/0.10.5/proc_installing-pipelines-operator-in-web-console.html

Reply:
@siamaksade - For the pipelines operator we have tried to follow a similar model as serverless given here - https://installing-pipelines--ocpdocs.netlify.com/openshift-enterprise/latest/serverless/installing_serverless/installing-openshift-serverless.html#installing-serverless-operator

Reply:
done